sdcm/nemesis.py

```diff
@@ -2398,6 +2399,165 @@ def disrupt_memory_stress(self):
         self.target_node.remoter.run(
             "stress-ng --vm-bytes $(awk '/MemTotal/{printf \"%d\\n\", $2 * 0.9;}' < /proc/meminfo)k --vm-keep -m 1 -t 100")
 
+    def disrupt_toggle_cdc_feature_properties_on_table(self):
+        """Manupulate cdc feature
+
+        randomly choose table and toggle cdc feature
+        with radomly generated cdc properties
+        """
+        self._set_current_disruption("ToggleCDCProperties")
+
+        if not self.tester.params.get("cdc_nemesis_enabled"):
```

Comment:
Why do we need this parameter?
In which cases we don't want to enable it?

Reply:
I think it's enough to check if the schema has CDC configured and if it does to run this nemesis.
The question is what happens if the first run of the nemesis will disable it, would the next run of the nemesis be able to detect that this table has/had cdc and will enable it?

Reply:
See my comments below on how to solve it...

Reply:
I added as we discussed, to have ability to totally disable run cdc nemesises. 
Yes, it at first run this nemesis disable cdc, then at next run it will enable it explicitly.

Reply:
Fixed